tests/components/component.jsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react'
+import React, {Component} from 'react'
 import TinyComponent from './tiny-component';
 
-export default class TestComponent extends React.Component {
-  constructor(){
-    super()
+export default class TestComponent extends Component {
+  constructor(...args) {
+    super(...args)
     this.state = {}
   }
 
-  render(){
+  render() {
     return <body>
```

Comment:
Body should be changed to div. we shouldn't render the body in react. If
you look at the react 14 branch, it's changed there because it caused a lot
of failing tests.
On Sat, Sep 26, 2015 at 02:59 Geoffroy Warin <notifications@github.com>
wrote:

> In tests/components/component.jsx
> <https://github.com/Legitcode/tests/pull/23#discussion_r40492257>:
>
> >      this.state = {}
> >    }
> >
> > -  render(){
> > +  render() {
> >      return <body>
>
> If changed to a div, this will break tests in find.jsx. Since this change
> would be somewhat unrelated to this pull request, maybe @zackify
> <https://github.com/zackify> should take a look a this [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Legitcode/tests/pull/23/files#r40492257>.
>